plugin/core/protocol.py

```diff
@@ -5912,6 +5912,18 @@ def semanticTokensFullDelta(cls, params: Mapping[str, Any], view: sublime.View)
     def semanticTokensRange(cls, params: Mapping[str, Any], view: sublime.View) -> 'Request':
         return Request("textDocument/semanticTokens/range", params, view)
 
+    @classmethod
+    def prepareCallHierarchy(cls, params: CallHierarchyPrepareParams, view: sublime.View) -> 'Request':
+        return Request("textDocument/prepareCallHierarchy", params, view)
+
+    @classmethod
+    def incomingCalls(cls, params: CallHierarchyIncomingCallsParams) -> 'Request':
+        return Request("callHierarchy/incomingCalls", params, None)
+
+    @classmethod
+    def outgoingCalls(cls, params: CallHierarchyOutgoingCallsParams) -> 'Request':
+        return Request("callHierarchy/outgoingCalls", params, None)
```

Comment:
It would probably be good to enable `progress` for those since this is user-initiated action and could in theory take a second to complete.